static/js/metrics/axis.js

```diff
@@ -0,0 +1,92 @@
+var X_TICK_FREQUENCY = 7;
```

Comment:
Not sure if rewriting this needs to be part of this PR or later improvements, but these could be `const`.

Reply:
Yeah I think these need moving in to the config too, will address in later PR